examples/game_of_life/static/index.html

```diff
@@ -2,11 +2,12 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>Yew • Game of Life</title>
+        <title>Yew Sample App</title>
         <link rel="stylesheet" href="styles.css">
-        <link rel="shortcut icon" href="favicon.ico">
+        <script type="module">
+            import init from "./wasm.js"
+            init()
+        </script>
     </head>
-    <body>
-        <script src="game_of_life.js"></script>
```

Comment:
What was wrong with this?

Reply:
Have a look in the build script. It expects for all examples the `index.html` template as shown on  [Build a Sample App - Yew Docs](https://yew.rs/docs/getting-started/build-a-sample-app):
```html
<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Yew Sample App</title>
        <script type="module">
            import init from "./wasm.js"
            init()
        </script>
    </head>
    <body></body>
</html>
```
I just inserted the missing `css` in the header.